package.json

```diff
@@ -50,8 +50,9 @@
     "@commitlint/config-conventional": "^8.1.0",
     "@react-native-community/bob": "^0.8.0",
     "@react-native-community/eslint-config": "^0.0.6",
-    "@react-navigation/core": "^3.5.1",
-    "@react-navigation/native": "^3.6.2",
+    "@react-navigation/core": "^5.2.0",
```

Comment:
you shouldn't use core directly. import everything from native

Reply:
I've updated it, thanks @satya164 👍